src/Bandwidth.Standard/Bandwidth.Standard.csproj

```
@@ -1,23 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo><!-- setting GenerateAssemblyInfo to false causes this bug https://github.com/dotnet/project-system/issues/3934 -->
     <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyName>Bandwidth.Standard</AssemblyName>
-    <PackageId>Bandwidth.Standard</PackageId>
-    <OutputType>Library</OutputType>
-    <Authors>OpenAPI</Authors>
-    <Company>OpenAPI</Company>
-    <AssemblyTitle>OpenAPI Library</AssemblyTitle>
-    <Description>A library generated from a OpenAPI doc</Description>
-    <Copyright>No Copyright</Copyright>
-    <RootNamespace>Bandwidth.Standard</RootNamespace>
-    <Version>1.0.0</Version>
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Bandwidth.Standard.xml</DocumentationFile>
-    <RepositoryUrl>https://github.com/GIT_USER_ID/GIT_REPO_ID.git</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <PackageReleaseNotes>Minor update</PackageReleaseNotes>
-    <PackageTags>Bandwidth</PackageTags>
+    <AssemblyName>Bandwidth.Sdk</AssemblyName>
+    <Version>1.0.0.0</Version>
+    <Authors></Authors>
+    <Owners></Owners>
+    <Product>Bandwidth.Standard</Product>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0.0</FileVersion>
+    <Description>.NET client library for the bandwidth API</Description>
+    <LangVersion>7.3</LangVersion>
```

```suggestion
    <LangVersion>7.3</LangVersion>
    <Copyright>Copyright © Bandwidth 2023</Copyright>
```
Lets also add copyright back, its in the other .csproj file